win32/CMakeLists.txt

```diff
@@ -19,15 +21,18 @@ set(pal_win32_c_files
     src/string_utils.c
     src/timer.c
     src/sync_win32.c
+    src/file_win32.c
 )
 
 FILE(GLOB pal_win32_md_files "devdoc/*.md")
 SOURCE_GROUP(devdoc FILES ${pal_win32_md_files})
 
 include_directories(${CMAKE_CURRENT_LIST_DIR}/inc)
+include_directories(../common/inc)
 
 add_library(pal_win32 ${pal_win32_h_files} ${pal_win32_c_files} ${pal_win32_md_files})
```

Comment:
>pal_win32_md_files [](start = 66, length = 18)

Might be nice to also add pal_common_md_files

Reply:
Coming up in next iteration

---
In reply to: [451914112](https://github.com/Azure/azure-c-pal/pull/7#discussion_r451914112) [](ancestors = 451914112)